src/test/resources/discovery/specification-sample-1.html

```diff
@@ -1,5 +1,5 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
-      xmlns:spec="http://www.w3.org/ns/spec#">
+      xmlns:schema="http://schema.org/" xmlns:spec="http://www.w3.org/ns/spec#">
```

Comment:
Might want to use the `prefix` attribute ( https://www.w3.org/TR/rdfa-core/#T-prefix ) instead. I tend to include prefixes on the `body` element.

Reply:
That's much better - I will do that in a subsequent update